test/commands/diff.test.ts

```diff
@@ -6,7 +6,7 @@ describe('diff', () => {
     test
       .stderr()
       .stdout()
-      .command(['diff', './test/specification.yml', './test/specification.yml', '--format=json'])
+      .command(['diff', './test/specification.yml', './test/specification.yml', '--format=json', '--no-log-diagnostics'])
```

Comment:
when I run `diff` successfully I do not want to see diagnostics by default. When I run `diff` I'm interested with `diff`, unless files that I run through diff have errors and `diff` cannot run because of it